From 8d8c6267cfc1477a6e476064e025f9d1ed283781 Mon Sep 17 00:00:00 2001
From: Matthew Feickert <matthew.feickert@cern.ch>
Date: Sat, 2 Nov 2024 07:38:10 +0000
Subject: [PATCH] convert print statements using 2to3

2to3 -f print -w Examples/UFO_MSSM/MSSM/write_param_card.py
---
 Examples/UFO_MSSM/MSSM/write_param_card.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Examples/UFO_MSSM/MSSM/write_param_card.py b/Examples/UFO_MSSM/MSSM/write_param_card.py
index 236c0e930..69f3dfb04 100644
--- a/Examples/UFO_MSSM/MSSM/write_param_card.py
+++ b/Examples/UFO_MSSM/MSSM/write_param_card.py
@@ -151,7 +151,7 @@ class ParamCardWriter(object):
         """ write qnumber """
         from particles import all_particles
         import particles
-        print particles.__file__
+        print(particles.__file__)
         text="""#===========================================================\n"""
         text += """# QUANTUM NUMBERS OF NEW STATE(S) (NON SM PDG CODE)\n"""
         text += """#===========================================================\n\n"""
@@ -177,5 +177,5 @@ class ParamCardWriter(object):
             
 if '__main__' == __name__:
     ParamCardWriter('./param_card.dat', generic=True)
-    print 'write ./param_card.dat'
+    print('write ./param_card.dat')
     
-- 
2.47.0

